exporters/otlp/internal/transform/span_test.go

```diff
@@ -360,3 +360,61 @@ func TestSpanData(t *testing.T) {
 		t.Fatalf("transformed span differs %v\n", diff)
 	}
 }
+
+func TestRootSpanData(t *testing.T) {
+	// Root span should have an empty ParentSpanId field.
+
+	// March 31, 2020 5:01:26 1234nanos (UTC)
+	startTime := time.Unix(1585674086, 1234)
+	endTime := startTime.Add(10 * time.Second)
+
+	sds := []*export.SpanData{
+		{
+			SpanContext: core.SpanContext{
+				TraceID: core.TraceID{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1},
+				SpanID:  core.SpanID{0, 0, 0, 0, 0, 0, 0, 1},
+			},
+			Name:      "invalid nil parent span ID",
+			StartTime: startTime,
+			EndTime:   endTime,
+		},
+		{
+			SpanContext: core.SpanContext{
+				TraceID: core.TraceID{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1},
+				SpanID:  core.SpanID{0, 0, 0, 0, 0, 0, 0, 1},
+			},
+			Name:      "empty parent span ID",
+			StartTime: startTime,
+			EndTime:   endTime,
+		},
+	}
```

Comment:
Aren't the above two identical?

Reply:
Yes, except for the name. I will refactor to be more targeted and remove the boilerplate.